### ISSUE TITLE
Typo

### ISSUE THREAD

**Issue by mKoonrad** [CodyHouse/cd-footnotes-changelog#2]
`<a href="#0" aria-lable="Download PDF">[PDF]</a>`

It should be aria-label

**Comment by claudia-romano** [comment]
Fixed, thanks!
